src/main/java/org/apache/ibatis/jdbc/AbstractSQL.java

```diff
@@ -374,13 +383,14 @@ public boolean isEmpty() {
     List<String> orderBy = new ArrayList<>();
     List<String> lastList = new ArrayList<>();
     List<String> columns = new ArrayList<>();
-    List<String> values = new ArrayList<>();
+    List<List<String>> values = new ArrayList<>();
```

Comment:
I prefer the `valuesList` rather than `values` in this case.

Reply:
I suggest to change `valuesList` rather than `valueList` because `valueList` same mean with `values`.

Reply:
have it fixed,so sad for that merge mistake.